tests/appsec/test_user_blocking_full_denylist.py

```diff
@@ -0,0 +1,52 @@
+from utils import context, released, coverage, interfaces, scenarios, weblog, bug, irrelevant, missing_feature
+
+
+@released(
+    cpp="?",
+    dotnet="2.30.0",
+    golang="1.48.0",
+    java="1.7.0",
+    nodejs="3.15.0",
+    php="0.86.3",
+    php_appsec="0.7.2",
+    python={"django-poc": "1.10", "flask-poc": "1.10", "*": "?"},
+    ruby="?",
+)
+@missing_feature(library="java", reason="/users endpoint is not implemented on java weblog")
+@bug(library="php")
+@bug(library="php_appsec")
+@bug(library="golang", reason="Doesn't work for versions <= 1.50.0")
```

Comment:
if it works for version higher than 1.50.0 then 

```suggestion
@bug(context.library <= "golang@1.50.0", reason="Doesn't work for versions <= 1.50.0")
```